.gitignore

```diff
@@ -1,6 +1,7 @@
 *.rbc
 *.swp
 .DS_Store
+/.bundle/gems
```

Comment:
I don't think bundler generates this folder by default. At least it doesn't for me. :thinking:

Reply:
Actually, I recently removed this from the official docker images since I thought that folder doesn't exist by default: https://github.com/docker-library/ruby/pull/306/commits/8f3650c2eaa81946c8bf878e3ed55efb0af7790f. I might be missing something here... :/

Reply:
d'oh, poked around and it looks like i configured it to do that 3 years ago(!) and forgot lmao

i'll close this and go add it to my global gitignore. :sweat_smile:

Reply:
d'oh, poked around and it looks like i configured it to do that 3 years ago(!) and forgot lmao

i'll close this and go add it to my global gitignore. :sweat_smile:

Reply:
Huh. I wonder if it's a config option that something suggests, perhaps?

Reply:
Ok, I was a bit worried that my commit to the docker library could break things for people. Good to know that it was a personal, old configuration and not some default :sweat_smile:.

Reply:
> Huh. I wonder if it's a config option that something suggests, perhaps?

Yeah, that was my concern, since it's too coincidental that the folders match... :man_shrugging:

Reply:
Specifically, this is my `.bundle/config`, and I think the last two lines are the relevant ones:

```
---
BUNDLE_GEM__COC: "true"
BUNDLE_GEM__MIT: "true"
BUNDLE_GEM__TEST: "rspec"
BUNDLE_PATH: ".bundle/gems"
BUNDLE_DISABLE_SHARED_GEMS: "true"
```

Reply:
Yeah... Well, this sounds like it belongs to you global gitignore indeed. But I'll investigate if maybe this is a side effect of the `disable_shared_gems` setting or somehing.

Reply:
Alright, I'll close this PR. :+1: